### PR TITLE
Add an item to check whether mTLS is enabled for a service

### PR DESCRIPTION
**Pull request by wattli**
Merge this one after https://github.com/istio/istio.github.io/pull/1972/files

### CODE REVIEW

File: content/help/faq/security/check-policy.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: How can I check whether mutual TLS is enabled for a service?
+weight: 11
+---
+
+Istioctl provides an 'authn' option to this purpose. You can do:
```
Comment (sdake) [inline content/help/faq/security/check-policy.md]: to this purpose->for this purpose

Comment (sdake) [inline content/help/faq/security/check-policy.md]: Istioctl-> The `istioctl` tool provides

File: content/help/faq/security/check-policy.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+{{< text bash >}}
+$ istioctl authn tls-check httpbin.default.svc.cluster.local
+HOST:PORT                                  STATUS     SERVER     CLIENT     AUTHN POLICY        DESTINATION RULE
+httpbin.default.svc.cluster.local:8080     OK         mTLS       mTLS       default/            default/default
+{{< /text >}}
+
+Refer to [Verifying TLS configuration](/docs/tasks/security/mutual-tls/#verifying-tls-configuration) for more information.
```
Comment (sdake) [inline content/help/faq/security/check-policy.md]: this is a bad link - but probably dependent on the other reorg taking place in the security section.

**Comment by sdake** [comment]
@wattli you will need to rebase after https://github.com/istio/istio.github.io/pull/1972/files merges.  The docs repo does not have automatic rebasing.

File: content/help/faq/security/check-policy.md (hunk 2 above)
Comment (wattli) [reply content/help/faq/security/check-policy.md]: This depends on https://github.com/istio/istio.github.io/pull/1972/files , i will rebase and ping back once that PR is merged.

**Comment by wattli** [comment]
@sdake , it's rebased, PTAL, thanks!

**Comment by wattli** [comment]
@geeknoid , PTAL, thanks

**Pull request merged**